src/main/java/dansplugins/simpleskills/api/preponderous/ponder/AbstractPonderPlugin.java

```diff
@@ -0,0 +1,23 @@
+package dansplugins.simpleskills.api.preponderous.ponder;
```

Comment:
We don't want the source code for the Ponder library to be in this repository. Would you please remove these files and put the dependencies folder back?

Reply:
that's ok but when run server without Ponder, plugins cannot (jar compile from src)

Reply:
oki oki ~~